gulp-tasks/utils/rollup-helper.js

```diff
@@ -12,6 +13,22 @@ module.exports = {
   getDefaultPlugins: (buildType) => {
     const plugins = [];
 
+    const babelConfig = {
+      presets: [['env', {
+        targets: {
+          // This corresponds to the version of Chroumium used by
+          // Samsung Internet 5.x, which is the minimum non-evergreen browser
+          // we currently support.
+          browsers: ['chrome >= 51'],
+        },
+        modules: false,
+      }]],
+      // The babelHelpers code will be injected into the workbox-core bundle.
```

Comment:
This message doesn't follow the next line referencing a file not in workbox-core.

I'm not following how this is all glued together.